uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document_test.js

```diff
@@ -25,6 +25,7 @@ define([
 
       data = {};
       files = {};
+      role = 'admin';
```

Comment:
Not testing! but the while instantiating the controller it expects me to pass role.